cmd/ci-operator-checkconfig/main.go

```diff
@@ -25,6 +25,10 @@ func main() {
 
 	seen := map[api.ImageStreamTagReference][]*config.Info{}
 	if err := config.OperateOnCIOperatorConfigDir(configDir, func(configuration *api.ReleaseBuildConfiguration, repoInfo *config.Info) error {
+		if err := validateCluster(configuration); err != nil {
```

Comment:
This should be a validation that we run on config load, not only here in `checkconfig`

Reply:
I bet it will break our integration tests.
Let me try.